feat(data): Making use of the new scanner info types (#70)
endless-sky/Endless-Sky-Delta: data/human/outfits.txt

	thumbnail "outfit/asteroid scanner"
	"mass" 6
	"outfit space" -6
	"asteroid scan power" 43
	description "This highly specialized scanning system can analyze the mineral composition of nearby asteroids and identify the ones that contain valuable materials."

outfit "Fuel Scanner"
	category "Systems"
	cost 64000
	thumbnail "outfit/tactical scanner"
	"mass" 2
	"outfit space" -2
	"fuel scan power" 35

> 	description "Knowing how much fuel your target has available is useless to most ship captains, but can be very valuable to both law enforcement and pirates. Sometimes the ocassional merchant uses one to determine if they are likely to be able to outlast someone in a long pursuit."

Reviewer: ```suggestion
	description "Knowing how much fuel your target has available is useless to most ship captains, but can be very valuable to both law enforcement and pirates. Sometimes the occasional merchant uses one to determine if they are likely to be able to outlast someone in a long pursuit."
```

outfit "Patrol Scanner"
	category "Systems"
	cost 64000
	thumbnail "outfit/tactical scanner"
	"mass" 2
	"outfit space" -2
	"crew scan power" 35
	"weapon scan power" 35

> 	description "Patrol ships, law enforcement, and immigration officials in the current stellar environemnt are generally concerned with two things: How many people are on board, and how far away does one need to be in order to stay safe. As such, these scanners are primarily found on ships who are responsible for tracking the flow of people and ocassionally conducting arrests or inspections."

Reviewer: ```suggestion
	description "Patrol ships, law enforcement, and immigration officials in the current stellar environment are generally concerned with two things: how many people are on board, and how far away does one need to be in order to stay safe. As such, these scanners are primarily found on ships who are responsible for tracking the flow of people and occasionally conducting arrests or inspections."
```

outfit "Rangefinder Scanner"
	category "Systems"
	cost 24000
	thumbnail "outfit/tactical scanner"
	"mass" 1
	"outfit space" -1
	"range finder power" 1

> 	description "While named a scanner for simplicity, this is more of an analysis and display module that utilizes standard system detection equipment to determine exact ranges to target ships. Once installed, the distance to the currently targeted ship will be display in the panel next to it."

Reviewer: ```suggestion
	description "While named a scanner for simplicity, this is more of an analysis and display module that utilizes standard system detection equipment to determine exact ranges to target ships. Once installed, the distance to the currently targeted ship will be displayed in the panel next to it."
```

outfit "Tactical Scanner"
	category "Systems"
	cost 144000
	thumbnail "outfit/tactical scanner"
	"mass" 4
	"outfit space" -4
	"velocity scan power" 35
	"energy scan power" 35
	description "A tactical scanner allows your ship to analyze the current velocity and energy levels of nearby targets, giving commanders useful information on the target's current movement. For centuries tactical scanners were only employed by the Navy, but recently Kraz Cybernetics has begun manufacturing them for use by militia ships."

outfit "Thermal Scanner"
	category "Systems"
	cost 64000
	thumbnail "outfit/tactical scanner"
	"mass" 2
	"outfit space" -2
	"thermal scan power" 35

> 	description "While thermal signatures have always been useful as a means of identifying and tracking ships, determining a ship's effective average thermal load was functionally useless until recently. With the development of the plasma canon and then the flamethrower, attempting to overheat a ship has become a viable combat technique. As such, engineers with the Free World have begun assembling these sensor packages to support the daring captains who choose to use this tactic."

Reviewer: ```suggestion
	description "While thermal signatures have always been useful as a means of identifying and tracking ships, determining a ship's effective average thermal load was functionally useless until recently. With the development of the plasma cannon and the flamethrower, attempting to overheat a ship has become a viable combat technique. As such, engineers with the Free World have begun assembling these sensor packages to support the daring captains who choose to use this tactic."
```

Author: I included the "then" in this case because the development of the plasma cannon and flamethrower were sequential and different things, not part of the same work. The former has been around for significantly longer than the flamethrower, while still being a recent invention, and I think having the "then" conveys this better.

Reviewer: True fact.
```suggestion
	description "While thermal signatures have always been useful as a means of identifying and tracking ships, determining a ship's effective average thermal load was functionally useless until recently. With the development of the plasma cannon and then the flamethrower, attempting to overheat a ship has become a viable combat technique. As such, engineers with the Free World have begun assembling these sensor packages to support the daring captains who choose to use this tactic."
```
